README.md

```diff
@@ -1,67 +1,67 @@
-﻿# Project Reunion
+﻿# Project Reunion - Calling all Windows apps!
 
-[![Feature Proposals](https://img.shields.io/github/issues/microsoft/projectreunion/feature%20proposal)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+proposal%22)
-[![Bugs](https://img.shields.io/github/issues/microsoft/projectreunion/bug)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+| Delight users with modern UI | Access new platform features | Backwards compatible | 
+|:--:|:--:|:--:|
+| <img src="https://docs.microsoft.com/media/illustrations/biztalk-get-started-get-started.svg" width=250 alt="Modern navigation"/><br>Powerful WinUI 3 UI | <img src="https://docs.microsoft.com/media/illustrations/biztalk-get-started-scenarios.svg" width=250 alt="Platform logo"/><br>Localization + power status + more<br> | <img src="https://docs.microsoft.com/media/illustrations/biztalk-host-integration-install-configure.svg" width=250 alt="Down-level logo"/><br>Down to Windows 10 1809 |
+
+...and more, **without having to rewrite your app!**
 
+* Use your current installer *(no requirement to use MSIX, but there are [reliability/security benefits to using MSIX](https://docs.microsoft.com/windows/msix/overview#key-features)!)*
+* 100% additive *(only add what you need, leave the rest of your app as-is)*
+* Works with Win32, WPF, WinForms, and UWP apps
 
-### What is Project Reunion?
-Project Reunion is our vision for unifying and evolving the Windows developer platform to make it easier
-to build great apps that work across all the Windows 10 versions and devices people use.
+Plus, **it's backwards compatible** to build 17763 (version 1809/October 2018 Update) of Windows 10!

```

Comment:
```suggestion
Plus, **it's backwards compatible** to Windows 10 October 2018 Update (build 17763 / version 1809)!
```

Nit: Lead with the official product name, then parenthetical the optional clarifying details.